h/util/logging_filters.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+"""Logging Filters."""
+
+import logging
+
+
+LEVELS = {"CRITICAL": 50,
+          "ERROR": 40,
+          "WARNING": 30,
+          "INFO": 20,
+          "DEBUG": 10,
+          "NOTSET": 0}
+
+
+class FilterExceptions(logging.Filter):
```

Comment:
Since the base class is named with a noun `logging.Filter`, I would suggest naming this as a noun rather than a verb - `ExceptionFilter`?